libs/go/aealite/protocols/dialogue.go

```diff
@@ -0,0 +1,437 @@
+package protocols
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"errors"
+	"fmt"
+)
+
+type Address string
+type Performative []string
+type SomeMessageType string
+
+const (
+	NONCE_BYTES_NB = 32
```

Comment:
my suggestion is to use camel case also for constants:

https://stackoverflow.com/a/22688926/7150172

Reply:
also, I find this comment is particularly meaningful:
https://stackoverflow.com/questions/22688906/go-naming-conventions-for-const#comment39214871_22688926